bug: Fixed a website compilation error caused by the <br> tag

### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/stoneatom/stonedb/blob/stonedb-5.7-dev/CONTRIBUTING.md).

### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Describe the problem

```js
Error:  Client bundle compiled with errors therefore further build is impossible.
SyntaxError: /home/runner/work/stonedb/stonedb/Docs/download.md: Expected corresponding JSX closing tag for <br>. (48:685)
  46 | <td parentName="tr" {...{"align":null}}><a parentName="td" {...{"href":"https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/stonedb-ce-5.7-v1.0.2.el7.x86_64.tar.gz"}}>{`Download`}</a></td>
  47 | <td parentName="tr" {...{"align":null}}>{`be54d4ebb7[45](https://github.com/stoneatom/stonedb/actions/runs/3938440572/jobs/6737132243#step:5:46)17bf09fbfe1806f6e2ab`}</td>
> [48](https://github.com/stoneatom/stonedb/actions/runs/3938440572/jobs/6737132243#step:5:49) | <td parentName="tr" {...{"align":null}}>{`1. You can `}<a parentName="td" {...{"href":"https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/stonedb-ce-5.7_v1.0.2.ubuntu.amd64.deb"}}>{`download the DEB package`}</a>{`；`}<br>{` 2. You can `}<a parentName="td" {...{"href":"https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/stonedb-ce-5.7-v1.0.2.el7.x86_64.rpm"}}>{`download the RPM package`}</a>{` and experience StoneDB installation in two minutes.；`}<br/>{` 3. You can `}<a parentName="td" {...{"href":"https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/shell.for.deploy.stonedb.as.replic.tar.gz"}}>{`download the Shell script`}</a></td>
     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              ^
  [49](https://github.com/stoneatom/stonedb/actions/runs/3938440572/jobs/6737132243#step:5:50) | </tr>
  [50](https://github.com/stoneatom/stonedb/actions/runs/3938440572/jobs/6737132243#step:5:51) | <tr parentName="tbody">
  [51](https://github.com/stoneatom/stonedb/actions/runs/3938440572/jobs/6737132243#step:5:52) | <td parentName="tr" {...{"align":null}}>{`Next Version`}</td>
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

### Expected behavior

_No response_

### How To Reproduce

I would delete this tag

### Environment

_No response_

### Are you interested in submitting a PR to solve the problem?

- [X] Yes, I will!

--- a/website/i18n/zh/docusaurus-plugin-content-docs/current/04-developer-guide/00-compiling-methods/compile-using-redhat7/compile-using-redhat7-for-57.md
+++ b/website/i18n/zh/docusaurus-plugin-content-docs/current/04-developer-guide/00-compiling-methods/compile-using-redhat7/compile-using-redhat7-for-57.md
@@ -148,7 +148,7 @@ cd boost_1_66_0
 ```
 boost 的安装路径可以根据实际情况指定，示例中的安装路径是 /usr/local/stonedb-boost。
 
-:::info
+:::tip
 在编译过程中，除非有关键字 "error" 报错自动退出，否则出现关键字 "warning"、"failed"是正常的，安装 boost 大概需要25分钟左右。
 :::
 ### 6. 安装 gtest
@@ -191,7 +191,7 @@ install_target=/stonedb57/install
 ###执行编译脚本
 sh stonedb_build.sh
 ```
-:::info
+:::tip
 如果是 CentOS/RedHat ，需要注释 os_dist 和 os_dist_release，并且修改 build_tag ，这是因为 "lsb_release -a" 返回的结果中，Distributor、Release、Codename 显示的是 n/a。注释 os_dist 和 os_dist_release 只会影响产生的日志名和 tar 包名，不会影响编译结果。
 :::
 ## 第五步：启动实例
@@ -238,13 +238,12 @@ chown -R mysql:mysql /stonedb57/install/my.cnf
 ### 3. 自动安装
 编译完成后，在安装目录下会自动生成 reinstall.sh、install.sh 和 my.cnf 文件，执行 reinstall.sh 就是创建目录、初始化实例和启动实例的过程。
 ```shell
-cd /stonedb56/install
+cd /stonedb57/install
 ./reinstall.sh
 ```
-:::info
+:::tip
 reinstall.sh 与 install.sh 的区别？<br />reinstall.sh 是自动化安装脚本，执行脚本的过程是创建目录、初始化实例和启动实例的过程，只在第一次使用，其他任何时候使用都会删除整个目录，重新初始化数据库。install.sh 是手动安装提供的示例脚本，用户可根据自定义的安装目录修改路径，然后执行脚本，执行脚本的过程也是创建目录、初始化实例和启动实例。以上两个脚本都只能在第一次使用。
 :::
-
 ### 4. 执行登录
 ```shell
 cat /stonedb57/install/log/mysqld.log |grep password
@@ -261,6 +260,6 @@ Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
 
 mysql> alter user 'root'@'localhost' identified by 'stonedb123';
 ```
-:::info
+:::tip
 管理员用户的临时密码在 mysqld.log 中，第一次登陆后需要修改管理员用户的密码。
 :::

--- a/website/i18n/zh/docusaurus-plugin-content-docs/current/download.md
+++ b/website/i18n/zh/docusaurus-plugin-content-docs/current/download.md
@@ -13,8 +13,8 @@ sidebar_position: 100
 | --- | --- | --- | --- | ---|
 | [5.6-v1.0.0](https://github.com/stoneatom/stonedb/releases/tag/stonedb-5.6-v1.0.0) | 2022-6-29 | [Download](https://static.stoneatom.com/stonedb-ce-5.6-v1.0.0.el7.x86_64.tar.gz) | 831f5799034fbb388f6ff26675b7951e |\|
 | [5.7-v1.0.0](https://github.com/stoneatom/stonedb/releases/tag/5.7-v1.0.0-GA) | 2022-8-31 |  [Download](https://static.stoneatom.com/custom/stonedb-ce-5.7-v1.0.0.el7.x86_64.tar.gz) | e23593b80b50e0118f59627216613a2f | \ |
-| [5.7-v1.0.1](https://github.com/stoneatom/stonedb/releases/tag/5.7-v1.0.1-GA) | 2022-10-24 |  [Download](https://static.stoneatom.com/custom/stonedb-ce-5.7-v1.0.1.el7.x86_64.tar.gz) | 73911e56ac89af5f568dd7e9de9e9af5 | 1. 可以[下载DEB包](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.1-GA/stonedb-ce-5.7-v1.0.1.debain.x86_64.tar.gz)；<br/>2. 如果您在编译的时候提示缺失依赖，您可以前往 Github  [**下载相关依赖**](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.1-GA/stonedb-lib.tar.gz) |
-| [5.7-v1.0.2](https://github.com/stoneatom/stonedb/releases/tag/5.7-v1.0.2-GA) | 2023-01-15 | [Download](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/stonedb-ce-5.7-v1.0.2.el7.x86_64.tar.gz) | be54d4ebb74517bf09fbfe1806f6e2ab  | 1. 可以[下载DEB包](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/stonedb-ce-5.7_v1.0.2.ubuntu.amd64.deb)；<br> 2. 可以[下载RPM包](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/stonedb-ce-5.7-v1.0.2.el7.x86_64.rpm)，体验两分钟极速安装StoneDB；<br/> 3. 可以下载自动化部署StoneDB备库的[Shell脚本](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/shell.for.deploy.stonedb.as.replic.tar.gz)|
+| [5.7-v1.0.1](https://github.com/stoneatom/stonedb/releases/tag/5.7-v1.0.1-GA) | 2022-10-24 |  [Download](https://static.stoneatom.com/custom/stonedb-ce-5.7-v1.0.1.el7.x86_64.tar.gz) | 73911e56ac89af5f568dd7e9de9e9af5 | 1. 可以[下载DEB包](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.1-GA/stonedb-ce-5.7-v1.0.1.debain.x86_64.tar.gz)；2. 如果您在编译的时候提示缺失依赖，您可以前往 Github  [**下载相关依赖**](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.1-GA/stonedb-lib.tar.gz) |
+| [5.7-v1.0.2](https://github.com/stoneatom/stonedb/releases/tag/5.7-v1.0.2-GA) | 2023-01-15 | [Download](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/stonedb-ce-5.7-v1.0.2.el7.x86_64.tar.gz) | be54d4ebb74517bf09fbfe1806f6e2ab  | 1. 可以[下载DEB包](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/stonedb-ce-5.7_v1.0.2.ubuntu.amd64.deb);2. 可以[下载RPM包](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/stonedb-ce-5.7-v1.0.2.el7.x86_64.rpm)，体验两分钟极速安装StoneDB；3.可以下载自动化部署StoneDB备库的[Shell脚本](https://github.com/stoneatom/stonedb/releases/download/5.7-v1.0.2-GA/shell.for.deploy.stonedb.as.replic.tar.gz)|
 | Next Version | Coming soon~  | Coming soon~  | Coming soon~ | \ |
 
 
@@ -23,5 +23,4 @@ sidebar_position: 100
 所有的下载包链接都导向了 Github 页面，如果您当前的网络无法正常访问 Github，可以前往 [Gitee](https://gitee.com/StoneDB/stonedb/releases/tag/5.7-v1.0.2-GA) 按需下载相关安装包。
 
 如果有更多问题，可以通过邮件联系我们：contact@stonedb.io
-
 :::

